### PR TITLE
doc / update installation gateway from source

### PR DESCRIPTION
**Pull request by glenntomas**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

- Updates and changes comes from QA @RC-13 and @PtrckM 

- Update `gateway.env` to `global_conf.yml`

image

- Update node version and gif
- Add nvm link in tip callout

image

- Update yarn installation and gif
- Update sample environment and gif

image

- Remove yarn run dev
- Update yarn debug
- Update yarn start and gif
- Update `.env` to `global_conf.yml`

image

- Updated the list of `ecr20_tokens_kovan.json` and the path of the file

image

**Tests performed by the developer**:

**Tips for QA testing**:

### CODE REVIEW

**Review by mifeng — APPROVED** [review]

**Comment by mifeng** [comment]
Since this is a correction to the published documentation only, we can merge this PR to `master` after 1 more approval. @dennisocana we should cherry-pick this commit onto `development`

**Comment by dennisocana** [comment]
@mifeng will do. Just an FYI similar to our hotfix process, we will revert this change from the `master` branch on the next release to avoid merge conflicts.

File: documentation/docs/installation/gateway.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ It downloads the scripts from GitHub, sets their correct permission and runs the
 
 By default, Gateway will install on port `5000` which Hummingbot will connect to. If the default port is not available, Gateway will find the next port number.
 
-4. The file `gateway.env` is created where your Gateway settings are saved.
+4. The file `global_conf.yml` is created where your Gateway settings are saved.
```
Comment (dennisocana) [inline documentation/docs/installation/gateway.md:52]: Incorrect file name.

File: documentation/docs/installation/gateway.md [hunk 2]
```diff
@@ -135,34 +135,31 @@ cd gateway-api
...
+yarn
+yarn build
 ```
 
 ![](/assets/img/gw_yarn.gif)
 
 ```bash
 # copy sample environment
-cp .env.example .env
+cp global_conf.yml.example global_conf.yml
```
Comment (dennisocana) [inline documentation/docs/installation/gateway.md:147]: Incorrect file name.

File: documentation/docs/installation/gateway.md [hunk 3]
```diff
@@ -135,34 +135,31 @@ cd gateway-api
...
 ![](/assets/img/gw_env.gif)
 
-- Edit `.env` file with your favorite text editor then save changes.
-- There are 3 ways to start the gateway
-
-```bash
-# run dev mode with hot reload on code changes
-yarn run dev
-```
+- Edit `global_conf.yml` file with your favorite text editor then save changes.
```
Comment (dennisocana) [inline documentation/docs/installation/gateway.md:152]: Incorrect file name.

**Review by dennisocana — CHANGES_REQUESTED** [review]

**Review by dennisocana — APPROVED** [review]

**Pull request merged**